### PR TITLE
Add Where op converter

### PR DESCRIPTION
**Pull request by disktnk**
t/o

### CODE REVIEW

File: onnx_chainer/context.py [hunk 1]
```diff
@@ -1,13 +1,29 @@
+import chainer
+
 from onnx_chainer import onnx_helper
 
 
 class Context(object):
+    """context of converter
```
Comment (shinh) [inline onnx_chainer/context.py]: nitpick: s/^c/C/ ?

File: onnx_chainer/context.py [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+
+    """
 
     def __init__(self, model):
         self.name_list = dict()
         for name, param in model.namedparams():
+            assert isinstance(param, chainer.Parameter)
             onnx_name = onnx_helper.cleanse_param_name(name)
             self.name_list[str(id(param))] = onnx_name
+            self.name_list[str(id(param.array))] = onnx_name
```
Comment (shinh) [inline onnx_chainer/context.py]: Maybe add a helper function like `set_array_name` and do something like

```
self.name_list[id(param)] = onnx_name
if isinstance(param, (chainer.Variable, chainer.Parameter)):
  self.name_list[id(param.array)] = onnx_name
```

?

Also, it'd be better to mention why we need ids of both `Variable` and `ndarray`. Perhaps we can say something like "some functions like F.where internally unwraps Variable"?

Comment (shinh) [reply onnx_chainer/context.py]: I'm also wondering if it's possible to use `id(a)` instead of `str(id(a))`.

Comment (disktnk) [reply onnx_chainer/context.py]: I fixed to aggregate `set_name` and rename `str(id(a))` to `id(a)`.

And I modified `Context` description to more clarify object of putting `ndarray` instance id. https://github.com/chainer/onnx-chainer/blob/d68fad0f7c09ad540e916b57d29a64595f2201ff/onnx_chainer/context.py#L12-L16

**Comment by disktnk** [comment]
ChainerCI, test this please

**Pull request merged**